File: gluecodium/src/test/resources/smoke/skip/output/android/com/example/smoke/SkipEnumeratorExplicitTag.java
/*

 *
 */

package com.example.smoke;


public enum SkipEnumeratorExplicitTag {
    ZERO(0),
    ONE(3),
    THREE(4);
    /**
     * @hidden
     */
    public final int value;

    SkipEnumeratorExplicitTag(final int value) {
        this.value = value;
    }
}

File: gluecodium/src/test/resources/smoke/skip/output/android-kotlin/com/example/smoke/SkipEnumeratorExplicitTag.kt
/*

 *
 */

@file:JvmName("SkipEnumeratorExplicitTag")

package com.example.smoke


enum class SkipEnumeratorExplicitTag(@JvmField val value: Int) {
    ZERO(0),
    ONE(3),
    THREE(4);
}
